cfpb/consumerfinance.gov
# Update expandables to handle long names, and update stylelint

Updates expandables to handle long expandable header names.

## Changes

- Update expandables and stylelint


## How to test this PR

1. yarn install and edit an expandable to show a long header and check that the toggle doesn't wrap

e.g.
financial well being http://localhost:8000/consumer-tools/financial-well-being/results/
iregs, at desktop/mobile http://localhost:8000/rules-policy/regulations/1002/
buying a home form explainers https://www.consumerfinance.gov/owning-a-home/loan-estimate/
rural or underserved app http://localhost:8000/rural-or-underserved-tool/#rural-or-underserved
